EIPS/eip-5630.md

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
@SamWilsn why is EIPW passing on this EIP? It should be throwing at least 5 errors (from what I've counted so far).

Reply:
you mean, the Walidator? I had to fix a bunch, but I think i got them all?

Reply:
No, there are some issues with this draft that EIP Walidator isn't catching but that need fixing. I don't know why EIPW isn't erroring. CC @SamWilsn

Reply:
k; please do; happy to address them.

Reply:
Hi @Pandapip1 @SamWilsn, any updates here? Thanks